cli/scorecard/cmd.go

```diff
@@ -20,11 +21,14 @@ func init() {
 
 	Cmd.Flags().StringVar(&flags.targetProjectID, "project", "", "Project to analyze (conflicts with --organization)")
 
-	Cmd.Flags().StringVar(&flags.bucketName, "bucket", "", "GCS bucket name for storing inventory")
-	Cmd.MarkFlagRequired("bucket")
+	Cmd.Flags().StringVar(&flags.inputPath, "input-path", "", "GCS bucket name (by default) OR local directory path (with --input-local option), for storing inventory")
```

Comment:
I think the way you had it before is better actually, having the same argument refer to two different options seems undesirable and harder to validate. Could we stick to two options: bucket or file-path?

Reply:
done in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/314/commits/68d7ddb5bf241cef64e2b2a0b42fe9f1a7472b41
(I call the 2nd param `dir-path` instead of `file-path` to indicate this parameter takes a directory)